index.html

```diff
@@ -1,44 +1,150 @@
-        <li><a href="/guide/intro/colors">Colors in pygame</a></li>
-    </ol>
-
-    <h2>Obscure but useful things</h2>
-    <ol>
-        <li><a href="/guide/obscure/pixelarray">PixelArray</a></li>
-        <li><a href="/guide/obscure/blend-modes">Blend modes</a></li>
-        <li><a href="/guide/obscure/set-mode-flags">pygame.display.set_mode flags</a></li>
-    </ol>
-</body>
+  <h1>Preface</h1>
+  <p>
+    "Where can I learn how to use pygame?" - one of the most commonly asked
+    questions that we get. But we always struggle to answer. We sometimes point
+    them to the docs and examples, but that isn't a comprehensive guide on
+    building good pygame-based applications. There are many other tutorials and
+    guides we have found, but we always find something lacking. There are many
+    comments we give and many nitpicks we have - so much so that some of us
+    gathered and decided to put all of them into a book. This book.
+  </p>
+
+  <h1>Prerequisites</h1>
+  <p>
+    So this book is meant for people who already know a bit of Python. Here's
+    what you need to know:
+  </p>
+  <ul>
+    <ul>
+      <li>
+        Basic control flow keywords such as - for, while, break, if, elif, else
+      </li>
+      <li>Functions</li>
+      <li>Understanding how types and instances work</li>
+      <li>Knowing how to access and modify attributes of objects</li>
+      <li>Familiarity with basic OOP(Object Oriented Programming)</li>
+      <li>
+        Knowing how to manage your packages (using pip, knowing whether you're
+        in a venv or not)
+      </li>
+    </ul>
+  </ul>
+
+  <p>
+    If you aren't familiar with the above, there are already some excellent
+    pre-existing Python resources out there that we suggest depending on your
+    situation
+  </p>
+  <h2>New to programming</h2>
+  <p>
+    If you're new to programming, here are some resources that are meant to
+    teach you from the ground up
+  </p>
+  <h3>Text resources:</h3>
+  <ul>
+    <li>Automate The Boring Stuff</li>
+    <li>A Byte of Python</li>
+  </ul>
+  <h3>Video resources:</h3>
+  <ul>
+    <li>Corey Schafer's Python basics series</li>
+    <li>Introduction to Python by Mike Dane</li>
+  </ul>
+
+  <h2>Familiar with programming</h2>
+  <p>
+    If you're already familiar with programming, you probably don't want
+    variables, loops, conditions and the like explained to you again, the
+    <a href="">Official Python Tutorial</a> acts as a schema to the language
+    while still being more expressive than documentation and covers the features
+    of the language nicely
+  </p>
+
+  <h1>Table of contents</h1>
+  <h2>Learning Pygame</h2>
+  <ol>
+    <li>
+      <a href="/guide/intro/chapter-1/">Chapter 01: What Is Pygame</a>
+    </li>
+    <li>
+      <a href="/guide/intro/chapter-2/">Chapter 02: Opening A Window</a>
+    </li>
+    <li>
+      <a href="/guide/intro/chapter-3/"
+        >Chapter 03: Surfaces And Positional Datatypes</a
+      >
+    </li>
```

Comment:
```suggestion
    <li>
      <a href="/guide/intro/chapter-3/">
        Chapter 03: Surfaces And Positional Datatypes
      </a>
    </li>
```

Reply:
I've actually reacted with a thumbsup for these, I just wanted to say this was the Prettier extension's formatting not mine :sweat_smile: . But I do agree it looks odd to me as well